crypto/threads_pthread.c

```diff
@@ -11,6 +11,10 @@
 
 #if defined(OPENSSL_THREADS) && !defined(CRYPTO_TDEBUG) && !defined(OPENSSL_SYS_WINDOWS)
 
+# if __STDC_VERSION__ >= 201112L && !defined(__STDC_NO_ATOMICS__)
+#  include <stdatomics.h>
```

Comment:
You're entirely correct.  I get confused by `__STDC_NO_ATOMICS__` (note ending S).  Fix coming up.